### PR TITLE
infra(unicorn): prefer-module

### PR DESCRIPTION
**Pull request by ST-DDT**
Ref: #2439

- #2439

---

Enables the [`unicorn/prefer-module`](https://github.com/sindresorhus/eslint-plugin-unicorn/blob/main/docs/rules/prefer-module.md) lint rule.

### CODE REVIEW

**Comment by ST-DDT** [comment]
As for `module_` see also https://github.com/sindresorhus/eslint-plugin-unicorn/issues/2208

- https://github.com/sindresorhus/eslint-plugin-unicorn/issues/2208
- https://github.com/sindresorhus/eslint-plugin-unicorn/pull/2209

Should we wait for it to be fixed?

File: scripts/generate-locales.ts [hunk 1]
```diff
@@ -359,8 +360,8 @@ async function normalizeLocaleFile(filePath: string, definitionKey: string) {
   }
 
   const fileContentPreData = fileContent.substring(0, compareIndex);
-  // eslint-disable-next-line @typescript-eslint/no-var-requires
-  const localeData = normalizeDataRecursive(require(filePath).default);
+  const fileImport = await import(`file:${filePath}`);
```
Comment (xDivisionByZerox) [inline scripts/generate-locales.ts:363]: Sorry if this is a stupid question but why is the "file:" protocal required here?

**Review by xDivisionByZerox — DISMISSED** [review]

File: scripts/generate-locales.ts (hunk 1 above)
Comment (ST-DDT) [reply scripts/generate-locales.ts:363]: Because it uses absolute paths. (and I think it gets confused with `c:` on windows.)

**Review by xDivisionByZerox — DISMISSED** [review]

**Review by xDivisionByZerox — APPROVED** [review]

**Comment by ST-DDT** [comment]
**Team Decision**

This is largely unchanged since creation and has been approved for more than a week (if you ignore the minor merge conflicts).

**Pull request merged**